src/app/components/Footer.tsx

```diff
@@ -181,12 +181,12 @@ function flattenSidebar(sidebar: SidebarItem[]) {
   const items: SidebarItem[] = []
 
   for (const item of sidebar) {
+    if (item.navigable || !item.items) {
```

Comment:
Maybe it would be better to check if the item has a `link` instead? That means we know for certain a page exists there, and we don't need the `navigable` property.

Reply:
Sounds good, I'll remove the new property and use `link`